pkg/network/additional_networks.go

```diff
@@ -30,7 +30,7 @@ func renderAdditionalNetworksCRD(manifestDir string) ([]*uns.Unstructured, error
 // renderRawCNIConfig returns the RawCNIConfig manifests
 func renderRawCNIConfig(conf *operv1.AdditionalNetworkDefinition, manifestDir string) ([]*uns.Unstructured, error) {
 	var err error
-	objs := []*uns.Unstructured{}
+	var objs []*uns.Unstructured
```

Comment:
There's no need for either declaration here; just use `:=` below. (Likewise in the other case, I think.)

Reply:
fixed in updated patchset